main/php_variables.c

```diff
@@ -615,24 +614,19 @@ PHPAPI void php_build_argv(char *s, zval *track_vars_array)
 			}
 		}
 	} else 	if (s && *s) {
-		ss = s;
-		while (ss) {
-			space = strchr(ss, '+');
-			if (space) {
-				*space = '\0';
-			}
+		const char* space;
```

Comment:
You can move this declaration into the while (combine it with the initialization).